src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Views/SeriesResultView.sql

```diff
@@ -0,0 +1,25 @@
+﻿/*************************************************************
+    Series View to be used for getting computed column response
+**************************************************************/
+IF NOT EXISTS 
+(
+    SELECT *
+    FROM    sys.views
+    WHERE   Name = 'SeriesResultView'
+)
+BEGIN
+    EXEC('CREATE VIEW dbo.SeriesResultView
+    WITH SCHEMABINDING
+    AS
+    SELECT  se.SeriesInstanceUid,
+            se.Modality,
+            se.PerformedProcedureStepStartDate,
+            se.ManufacturerModelName,
+            (SELECT SUM(1)
+            FROM dbo.Instance i 
+            WHERE se.SeriesKey = i.SeriesKey) AS NumberofSeriesRelatedInstances,
```

Comment:
Based on our schema,  we can't have same series in different partition. But to be consistent should we include partitionKey?

Reply:
Same comment as before. There is no matching index. I will fix all the indexes in the next PR<>